eZ Platform/Admin UI/Field Type Edit/Requirements.md

```diff
@@ -0,0 +1,45 @@
+# Content editing
+
+
+# Objective
+
+"As an eZ Engineer, I want...
+
+To create a functional, experimental implementation of a refactoring of the Field Type API in eZ Platform supporting full CRUD (create, read, update and delete) for content creation and editing. The purpose of the research is to find a common way to define the user interface using different technologies."
+
+Underlying goals are:
+
+* Simplify how to create new custom Field Type
+* Reduce errors by limiting the number of places UI definitions are done
+* Increase developer velocity when developing and maintaining Field Types
+
+# Firm
+
+The prototype *must* for the following interfaces:
+* Twig based content editing
+* A remote website (e.g. Angular)
+* An App using REST API (e.g.React Native)
+
+Provide examples that:
+
+* Demonstrate how we can define input validation
+* Demonstrate how we can define input UI commonly
+    * Preferably ending with auto generating on the different platforms a working default
+* Demonstrate how we can override UI components where platform specific implementation would be more desirable
+
+
+# Preferred
+
+The prototype *should*:
+* Demonstrate how Symfony forms (repo-forms) can be used
```

Comment:
I guess the question lies in scope here, current implementation is limited so there are several things that needs to be look into in order to prove it can be used for content editing.